src/fmu/dataio/dataio.py

```diff
@@ -812,47 +812,31 @@ class InitializeCase:  # pylint: disable=too-few-public-methods
             some predefined main level keys. If config is None or the env variable
             FMU_GLOBAL_CONFIG pointing to a file is provided, then it will attempt to
             parse that file instead.
-        rootfolder: To override the automatic and actual ``rootpath``. Absolute path to
-            the case root, including case name. If not provided (which is not
-            recommended), the rootpath will be attempted parsed from the file structure
-            or by other means.
+        rootfolder: Absolute path to the case root, including case name.
         casename: Name of case (experiment)
         caseuser: Username provided
-        restart_from: ID of eventual restart (deprecated)
-        description: Description text as string or list of strings.
+        description (Optional): Description text as string or list of strings.
```

Comment:
Should we get rid of description?

Reply:
Perhaps. I think the description as such needs to be there/somewhere, but we don't really have a good interface for inserting it anywhere before or during runtime of FMU. In Sumo, the description is mandatory to be allowed to "keep" the case. I.e. we try to force people to give context that we are not capturing in metadata.

![image](https://github.com/equinor/fmu-dataio/assets/35033325/0fa63a53-237d-494e-bf34-9514d1736505)

One could imagine that during FMU runtime, this would be something best facilitated via ERT.

It is also unclear to me why this is a list, and not just a string.

Another point to be made is that this is a 100% free-text field. It is _probably_ just a question of time before someone starts to put special strings into the description and then program on top of it. So that we want to avoid.

Separate issue?

Reply:
Yes, I think this is a separate issue... not clear how it's best solved, but pointing to ERT probably seems the most sane to me.